README.md

```diff
@@ -1,47 +1,106 @@
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [1.x]    | **Maintenance LTS** |  2018-08-24     | 2018-11-20            | March 2019  | 1.x
 | [2.x]    | **Current Release** |  2018-11-20     |                       |             | 2.x
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+- [Troubleshooting](#troubleshooting)
+- [Tests](#tests)
 
-## Continuous Integrations 
+## Usage
 
-### Travis CI 
-[![Build Status](https://travis-ci.org/vtex-apps/category-menu.svg?branch=master)](https://travis-ci.org/vtex-apps/category-menu)
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
 
-## Usage
+To use this app you need to add it in your dependencies in the `manifest.json` file.
 
-TODO
+```json
+  dependencies: {
+    "vtex.category-menu": "2.x"
+  }
+```
 
-### Running Tests
+Then, add `category-menu` block into your app theme, like we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
 
-To run the test suit, type the following in the terminal, inside the folder `react`
+### Blocks API
+:construction: :construction: :construction:
 
-```sh
-$ npm t     # this
-$ yarn test # or this one
-```
+This app has an interface that describes what rules must be implemented by a block when you want to use the category-menu block.
 
-To update the tests snapshots use
+```json
+  "category-menu": {
+    "component": "index"
+  }
+}
+```
 
-```sh
-$ npm t -- -u  # like this
-$ yarn test -u # or this
+#### Configuration 
+Through the Storefront you can change the behavior and interface of `CategoryMenu`. But, you can also make adjusts in your theme app, like Dreamstore does.
+
+| Prop name          | Type       | Description                                                                 |
+| ------------------ | ---------- | --------------------------------------------------------------------------- |
+| `showPromotionCategory`                      | `Boolean`   | Shows the promotion category in menu               |
+| `showGiftCategory`          | `Boolean`  | Shows the gift category in menu                               |
+| `showAllDepartments`              | `Boolean`  | Shows all departments category in menu                              |
+| `menuDisposition`                   | `Enum`  | Indicates the disposition of the menu on the screen (left, center, right)                                     |
```

Comment:
What are the enum options?

Reply:
They are indicated in the description:  (left, center, right). The "old" doc used this style for Enum types.

Reply:
@brunojdo what you think if I do this? `Enum (left, center, right)`